### DISCUSSION TITLE
Thank you for this project

### DISCUSSION THREAD

**Discussion by auyongxyz** [obsidian-html/obsidian-html#213] [General]
Thank you @dwrolvink. It is a wonderful utility.

I'd like to buy you a coffee if you have any preferred way of accepting a token donation.

**Comment by dwrolvink** [comment]
Thank you, I appreciate it :smile: 

This is my first open source contribution after a lifetime of using FOSS software, so I'm just happy to be giving something back.

But for those who want to buy me a coffee regardless, they can now do so via this link https://www.buymeacoffee.com/dwrolvink

Cheers!

**Comment by dr-code** [comment]
I’d like to second jmowiki.
This is fantastic, and underrecognized to be honest. 

I’d like to buy you a coffee as well! Thanks again!
